README.md

```diff
@@ -177,6 +177,57 @@ storage.directory=/var/lib/janusgraph/data
 index.search.directory=/var/lib/janusgraph/index
 ```
 
+#### Gremlin Server Environment Variable Syntax
+Environment Variables that start with the prefix `gremlinserver.*` or `gremlinserver%d.*` are used to edit the base gremlin-server.yaml file. The text after the prefix in the environment variable name should follow a specific syntax. This syntax is implemented using the [yq][YQ_GITHUB] write and delete commands and the [yq documentation][YQ_DOC] can be used as a reference for this syntax. Secondly, the value of the environment variable will be used to set the value of the key specified in the environment variable name.
+
+Let's take a look at a few examples:
+
+##### Nested Properties
+
+```text
+$ docker run --rm -it -e gremlinserver.graphs.ConfigurationManagementGraph=conf/JanusGraph-configurationmanagement.properties janusgraph/janusgraph:0.4.0 janusgraph show-config
+...
+graphs:
+  graph: conf/gremlin-server/janusgraph-cql-es-server.properties
+  ConfigurationManagementGraph: conf/JanusGraph-configurationmanagement.properties
+scriptEngines:
+...
+```
+##### Delete a component
+To delete a component append %d to the 'gremlinserver.' prefix before the closing dot and then select the component following the prefix. Don't forget the trailing '='.
```

Comment:
Wrap long line.

Reply:
> I added a few more nitpick comments regarding the formatting of the `README.md`.

@FlorianHockmann Thanks I didn't realize that. I made those final changes you mentioned. Let me know if there is you see anything else.